Makefile

```diff
@@ -33,3 +33,12 @@ test: ## Run tests.
 
 vet: # Vet the code
 	go vet $(CHECK_FILES)
+
```

Comment:
I also had to run the following to generate and serve the swagger docs
```
make deps
go mod tidy

go get -u github.com/go-swagger/go-swagger/cmd/swagger
go install github.com/go-swagger/go-swagger/cmd/swagger
```

I didn't go back to check if all of these are actually needed but I think it would be good to add those that are to the `make deps` command. Adding a section to the `README.md` on how to generate the api docs would also be good for new contributors.